tests/LINEBot/Util/DummyHttpClient.php

```diff
@@ -20,6 +20,7 @@
 
 use LINE\LINEBot\HTTPClient;
 use LINE\LINEBot\Response;
+use phpDocumentor\Reflection\Types\Integer;
```

Comment:
is this used?

Reply:
What is this for?

Reply:
My editor automaticlly added 'use phpDocumentor\Reflection\Types\Integer;' when I wrote  below in first
`__construct(TestCase $testRunner, \Closure $mock, int $statusCode = 200)`

I removed type definition 'int' because php5 not support int type hinting.
But I forgot removing use 'phpDocumentor\Reflection\Types\Integer'...

Reply:
'use phpDocumentor\Reflection\Types\Integer;' is not needed so I remove this.